lib/src/main/java/com/yalantis/contextmenu/lib/MenuObject.kt

```diff
@@ -89,41 +89,41 @@ open class MenuObject(var title: String = "") : Parcelable {
 
     override fun describeContents(): Int = 0
 
-    fun setBgDrawable(drawable: ColorDrawable) {
+    open fun setBgDrawable(drawable: ColorDrawable) {
```

Comment:
All the stuff below should not be open I guess. If the user will not override them properly the will have broken functionality